api/apps/api/src/modules/projects/projects-crud.service.ts

```diff
@@ -136,6 +136,7 @@ export class ProjectsCrudService extends AppBaseService<
      */
     const project = await super.setDataCreate(create, info);
     project.createdBy = info?.authenticatedUser?.id!;
+    project.planningAreaGeometryId = create.planningAreaId;
```

Comment:
🙈 is geometry the same as area?

Reply:
I didn't want to change the api contract, and it looked like setting `planningAreaId` is used to set a custom planning area, whereas `planningAreaId` is a generic calculated field in the entity, and `planningAreaGeometryId` is a actual column in the entity representing the assigned custom planning area